source/envs/env2048.py

```
@@ -0,0 +1,540 @@
+        invalid_move_penalty: float = 0.0,
+        target_tile: int = 16384,
+        end_on_win: bool = False,
+        spawn_prob_4: float = 0.1,
+        max_tile_exp: int = 17,
+        # 렌더링/UX
+        tile_px: int = 100,
+        gap_px: int = 10,
+        fps: int = 30,
+    ):
+        super().__init__()
+        assert size >= 2
```

Ruff (0.12.2)

Use of `assert` detected

(S101)